partials/seo.php

```diff
@@ -29,7 +29,7 @@
 } else if (!empty($ogImage)) {
   echo '<meta property="og:image" content="' . $ogImage[0] . '" />';
 } else {
-  echo '<meta property="og:image" content="' . get_stylesheet_directory_uri() . '/img/dist/favicon.png" />';
+  echo '<meta property="og:image" content="' . get_template_directory_uri() . '/dist/img/favicon.png" />';
```

Comment:
Is this wrong in the boiler?

Reply:
yes. the whole seo partial is pretty wrong